src/components/SuggestedItem/SuggestedItem.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import "./SuggestedItem.css";
+
+export default props => {
+  const {
+    // flag,
+    handleClick,
+    image,
+    name,
+    pack,
+    index
+  } = props;
+  const toBePacked = pack ? "item--font--packed" : "item--font--unpacked";
+  const borderColor = pack ? " item--border--packed " : " item--border--unpacked ";
+  return (
+    <div
+      className={toBePacked + borderColor +"  item p-0 m-1 rounded"}
+      onClick={handleClick("bundle", index)}
+    >
+      <div className="container">
+        <div className={"row align-items-center justify-content-center"}>
+          <div className="col-12 px-1 pt-1  text-left div--icon--size">
+            {/* ICONS */}
+            {pack ? (
+              <i className="far fa-check-circle align-top item--icon--bundled" />
+            ) : (
+              <i className="far fa-circle align-top item--icon--unbundled" />
+            )}
+          </div>
+          <div>
+            {/* ITEM IMAGE */}
+            <img src={image} className="item--image--size rounded" alt={name} />
+          </div>
+        </div>
+      </div>
+      <div className="col-12">
+        {/* Item Name */}
+        <div className={"row align-items-center justify-content-center"}>
+          <span className="item--title--font">{name}</span>
+        </div>
+      </div>
+      <div className="col-12" onClick={handleClick("select", index)}>
```

Comment:
> `<div ... onClick={handleClick("select", index)}>`

Normally, if you have an `onClick` handler for something important, you would want to put it on an `input` or `button`. Those elements are made for handling clicks and will also let you tab to them and trigger the event with space/enter keys. It'd be better if you switched this to a `button`.